webpack.config.js

```diff
@@ -103,40 +103,15 @@ const webpackConfig = {
 					amd: false,
 				},
 			},
-			{
-				test: /\.jsx?$/,
-				exclude: /node_modules/,
-				use: {
-					loader: 'babel-loader?cacheDirectory',
-					options: {
-						presets: [ '@wordpress/babel-preset-default' ],
-					},
-				},
-			},
 			{
 				test: /\.js?$/,
+				exclude: /node_modules/,
 				use: {
 					loader: 'babel-loader',
 					options: {
-						presets: [
-							[
-								'@babel/preset-env',
-								{ loose: true, modules: 'commonjs' },
-							],
-						],
-						plugins: [ 'transform-es2015-template-literals' ],
+						presets: [ '@wordpress/babel-preset-default' ],
```

Comment:
This covers preset env and others. preset env handles template literals as well so no need for the plugin.